src/routes/(protected)/api/tvdb/search/+server.ts

```
@@ -0,0 +1,183 @@
+import type { RequestHandler } from "./$types";
+import { json, error } from "@sveltejs/kit";
+import providers from "$lib/providers";
+
+/**
+ * Apply server-side filters to TVDB results
+ */
+function applyServerFilters(items: any[], filters: Record<string, any>): any[] {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)